### ISSUE TITLE
[Accessibility] The only indication that a device is connected that the icon turns green in the panel

### ISSUE THREAD

**Issue by drwill-ms** [Microsoft/vscode-azure-iot-edge#362]
- VSCode Version: 1.31.0
- OS Version: 10
- Extension Version: 1.7.0

Steps to Reproduce:

1. Create and attached devices and then have them send data
2. Note the only indication the device is connected and sending data is that the icon is green. This breaks accessibility rules as color alone cannot solely indicate a thing.
![image](https://user-images.githubusercontent.com/14119884/52591374-b26ae500-2df8-11e9-95a6-b3457ca7c5e2.png)

Does this issue occur when all other extensions are disabled?: Yes

**Comment by drwill-ms** [comment]
#AzureIoTE2E

**Comment by adashen** [comment]
The functionality is from vscode iothub toolkit. Issue has been created in toolkit repo
https://github.com/Microsoft/vscode-azure-iot-toolkit/issues/269 mark external dependency here

**Comment by Fornax1** [comment]
I've notice the same, when I'm connected with MXchip. This is a good thing.

**Comment by formulahendry** [comment]
![image](https://user-images.githubusercontent.com/1050213/54108882-0d9fef80-4418-11e9-8b20-57107d348c6e.png)
Released in v2.4.0, if device is connected, we will also add a 'Connected' text.
